si_ps_conv_fastcorrsApp/Db/kicks.template

```diff
@@ -0,0 +1,219 @@
+    field(FLNK, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVH PP")
+}
+
+record(ai, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVH") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DTYP, "Python Device")
+    field(SCAN, "5 second")
+    field(INP, "@curr2strn $(SEC) $(DIP) $(ORI) Current-DRVH")
+    field(FLNK, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVH")
+}
+
+record(ao, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVH") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DOL, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVH")
+    field(OMSL, "closed_loop")
+    field(OUT, "SI-$(SEC)$(DIP):PS-$(ORI):Kick-SP.DRVH")
+}
+
+record(sseq, "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):Current-RB") {
+    field(PINI, "YES")
+    field(DOL1, "SI-$(SEC)$(DIP):PS-$(ORI):Current-RB CP")
+    field(LNK1, "copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-RB PP")
+}
+
+record(bi, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):CurrentRBCnct-Mon") {
+    field(PINI, "YES")
+    field(SCAN, "1 second")
+    field(INP , "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):Current-RB.DOL1V")
+    field(ZNAM, "Ext PV Not Connected")
+    field(ZSV , "MAJOR")
+    field(ONAM, "Ext PV OK")
+}
+
+record(sseq, "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):CurrentRef-Mon") {
+    field(PINI, "YES")
+    field(DOL1, "SI-$(SEC)$(DIP):PS-$(ORI):CurrentRef-Mon CP")
+    field(LNK1, "copy-SI-$(SEC)$(DIP):PS-$(ORI):CurrentRef-Mon PP")
+}
+
+record(bi, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):CurrentRefCnct-Mon") {
+    field(PINI, "YES")
+    field(SCAN, "1 second")
+    field(INP , "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):CurrentRef-Mon.DOL1V")
+    field(ZNAM, "Ext PV Not Connected")
+    field(ZSV , "MAJOR")
+    field(ONAM, "Ext PV OK")
+}
+
+record(sseq, "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):FOFBAcc-Mon") {
+    field(PINI, "YES")
+    field(DOL1, "SI-$(SEC)$(DIP):PS-$(ORI):FOFBAcc-Mon CP")
+    field(LNK1, "copy-SI-$(SEC)$(DIP):PS-$(ORI):FOFBAcc-Mon PP")
+}
+
+record(bi, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):FOFBAccCnct-Mon") {
+    field(PINI, "YES")
+    field(SCAN, "1 second")
+    field(INP , "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):FOFBAcc-Mon.DOL1V")
+    field(ZNAM, "Ext PV Not Connected")
+    field(ZSV , "MAJOR")
+    field(ONAM, "Ext PV OK")
+}
+
+record(sseq, "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):Current-Mon") {
+    field(PINI, "YES")
+    field(DOL1, "SI-$(SEC)$(DIP):PS-$(ORI):Current-Mon CP")
+    field(LNK1, "copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-Mon PP")
+}
+
+record(bi, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):CurrentMonCnct-Mon") {
+    field(PINI, "YES")
+    field(SCAN, "1 second")
+    field(INP , "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):Current-Mon.DOL1V")
+    field(ZNAM, "Ext PV Not Connected")
+    field(ZSV , "MAJOR")
+    field(ONAM, "Ext PV OK")
+}
+
+record(sseq, "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP") {
+    field(PINI, "YES")
+    field(DOL1, "SI-$(SEC)$(DIP):PS-$(ORI):Current-SP CP")
+    field(LNK1, "copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP PP")
+}
```

Comment:
Either this record or `copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP` are in the wrong spot, since these records depend on one another and should be easy to view simultaneously.